nntrainer/nntrainer_log.h

```diff
@@ -40,42 +40,49 @@
 #elif defined(__ANDROID__)
 #include <android/log.h>
 
-#define ml_logi(...) \
+#define ml_logi(...)                                                           \
   __android_log_print(ANDROID_LOG_INFO, TAG_NAME, __VA_ARGS__)
 
-#define ml_logw(...) \
+#define ml_logw(...)                                                           \
   __android_log_print(ANDROID_LOG_WARN, TAG_NAME, __VA_ARGS__)
 
-#define ml_loge(...) \
+#define ml_loge(...)                                                           \
   __android_log_print(ANDROID_LOG_ERROR, TAG_NAME, __VA_ARGS__)
 
-#define ml_logd(...) \
+#define ml_logd(...)                                                           \
   __android_log_print(ANDROID_LOG_DEBUG, TAG_NAME, __VA_ARGS__)
 
+#elif defined(_WIN32)
+
+#define ml_logi(format, ...)
+#define ml_logw(format, ...)
+#define ml_loge(format, ...)
+#define ml_logd(format, ...)
```

Comment:
Except for this code, the code diff was generated due to line breaks and indentation alignment.